handout/forms/template-driven_forms.md

```diff
@@ -0,0 +1,58 @@
+# Template-Driven Forms
+
+The most straightforward approach to building forms in Angular 2 is to take advantage of the directives which have already been made for you.
+
+First, consider a typical form:
+
+```html
+<form method="POST" action="/register" id="SignupForm">
+  <label for="email">Email</label>
+  <input type="text" name="email" id="email">
+
+  <label for="password">Password</label>
+  <input type="password" name="password" id="password">
+
+  <button type="submit">Sign Up</button>
+</form>
+```
+
+The simplest method for dealing with forms in Angular 2 is not much more complicated than this.
+That's because Angular has already provided you a `form` directive (and friends), which operates under the covers.
```

Comment:
It might be worth mentioning that Angular is actually implementing the `ngFor` directive to any `form` element found in a template. That will help you explain why the syntax `#signupForm="ngForm"` below. It's a way to export a reference to that particular instance of the `ngForm` directive.